src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -170,38 +188,71 @@ public Object createModule(String name) throws CheckstyleException {
 
     /**
      * Create object with the help of the supplied map.
-     * @param name name of module.
-     * @param map the supplied map.
-     * @return instance of module if it is found in modules map.
-     * @throws CheckstyleException if the class fails to instantiate.
+     * @param name name of module
+     * @param map the supplied map
```

Comment:
> see there are no period in other javadoc tag in PackageObjectFactory

I see them in a few places like `PackageObjectFactory.joinPackageNamesWithClassName`. 
Also looking at the history, you added the `map` parameter and the period on the end of that one. :)

Seeing as we have no styles placed on the Javadocs, I don't see removing the periods as necessary and just superfluous as someone else will come around and add them back in.
Please just add the periods back for now. Sometime we will enforce this through a Javadoc style guide.